.buildkite/scripts/run_ppo_test.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+# SPDX-License-Identifier: Apache-2.0
+# Setup Verl + vLLM environment, run GSM8K Qwen0.5B ppo example, then test with vLLM
+
+set -euo pipefail
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+REPO_ROOT="$(cd "${SCRIPT_DIR}/../.." && pwd)"
+
+VERL_REPO="https://github.com/volcengine/verl.git"
+VERL_BRANCH="main"
+VERL_DIR="${REPO_ROOT}/verl"
+TARGET_DIR="${VERL_DIR}/examples/data_preprocess"
+MODEL_ID=Qwen/Qwen2.5-0.5B-Instruct
+MODEL_DIR="${VERL_DIR}/models/Qwen2.5-0.5B-Instruct"
+train_epochs=2
+data_dir="${VERL_DIR}/gsm8k"
+n_gpus_per_node=8
```

Comment:
Does it require 8 GPUs? We have some 4 GPU H100 runners already that we could run this on nightly